### ISSUE TITLE
Caprine cannot detect built-in webcam

### ISSUE THREAD

**Issue by trmaphi** [sindresorhus/caprine#568]
Tested webcam with cheese

System info
-------------- 
OS: Ubuntu 18.04.1 LTS x86_64 
Host: 42361S9 ThinkPad T420 
Kernel: 4.15.0-32-generic 
Uptime: 3 hours, 28 mins 
Packages: 1760 
Shell: bash 4.4.19 
Resolution: 1920x1080 
DE: GNOME 3.28.2 
WM: GNOME Shell 
WM Theme: Adwaita 
Theme: Materia-light [GTK2/3] 
Icons: Flat-Remix-Light [GTK2/3] 
Terminal: gnome-terminal 
CPU: Intel i7-2620M (4) @ 3.400GHz 
GPU: Intel 2nd Generation Core Proce 
Memory: 3737MiB / 7749MiB

**Comment by dbisk** [comment]
I was also on Ubuntu 18.04.1 and it worked fine. Does it work if you use facebook messenger from a browser?

**Comment by trmaphi** [comment]
Facebook messenger works normaly with the webcam.

**Comment by trmaphi** [comment]
@dbisk How do you install the caprine? I installed it via snap.
I did build from the source the camera work normal!

**Comment by dbisk** [comment]
I installed it using snap. Mine seems to work fine though, so I'm not sure if the reason is snap...

**Comment by trmaphi** [comment]
@dbisk  I had a workaround, that's put `--classic`. The reason is security confinement of [snap](https://docs.snapcraft.io/reference/snap-command#command-options-1)
```
snap install --classic caprine
```

**Comment by dbisk** [comment]
@trmaphi Yes, you're correct. I forgot I installed with the `--classic` option.
